elements/lisk-p2p/src/p2p_request.ts

```diff
@@ -102,4 +104,17 @@ export class P2PRequest {
 	public error(responseError: Error): void {
 		this._respondCallback(responseError);
 	}
+
+	// eslint-disable-next-line class-methods-use-this
+	private _getBinaryData(data: unknown): string | undefined {
```

Comment:
maybe have more specific name, like `getBinaryBufferData` as in `base.ts` we have the same func but returning string